addons/survey/views/survey_templates.xml

```diff
@@ -88,7 +88,7 @@
                 class="d-flex flex-grow-1"
                 t-att-data-answer-token="answer.access_token"
                 t-att-data-survey-token="survey.access_token"
-                t-att-data-users-can-go-back="survey.users_can_go_back"
+                t-att-data-users-can-go-back="survey.users_can_go_back and not answer.is_session_answer"
```

Comment:
Would be better to use: `survey._can_go_back(answer, question)` here IMO

Reply:
indeed, if question is null (because we are in section mode), this will work anyway. :)

Reply:
Well, not really, because _can_go_back will be evaluated at first rendering but navigating from one page (or question) to another will not re-evaluate this. and it will always be false (due to state == "in_progress" condition -> when beginning the survey, state = new)
we just need to know if the Go Back option is active and is we are not in session. 
Can_go_back does a little bit more.